_inc/client/writing/index.jsx

```diff
@@ -98,7 +98,7 @@ export const Writing = ( props ) => {
 		if ( isVideoPress ) {
 			if ( fetchingSiteData ) {
 				toggle = '';
-			} else if ( ! sitePlan || 'jetpack_free' === sitePlan.product_slug || /jetpack_personal*/.test( sitePlan.product_slug ) ) {
+			} else if ( ! sitePlan || 'free_plan' === sitePlan.product_slug || 'jetpack_free' === sitePlan.product_slug || /jetpack_personal*/.test( sitePlan.product_slug ) ) {
```

Comment:
we should check for 'personal-bundle' here too, though it shouldn't come up at this point